free-2.md

```diff
@@ -11,41 +11,58 @@ _What need is met by your idea? Who are you helping? What is that person's
+
 ### Operation (semantics)
 
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+When the program runs it would create a base data structure to hold data for a particular game be that a football game or soccer game, each structure would be diffrent according to each sport. This data structure would be added to as the user utalizes the laguage, adding in individual players and their stats. After a command by the user is given, a success message can be delivered, additionally after the game has concluded the user will be able to see a represention of each individual players stats for the game as well as an entire team stats collection. This would be done by internal processing that calculates specific sports stats such as offensive yards per game. with additional features this dsl may also hav ethe ability to grab particular stats from past games for comparisions. Errors should be comunicted with error messages to the user letting them know if a particular command or stat was not stored.
+
 ### Expressiveness
 
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to start a new game log of stats that is easy to input for the user as well as save them time. The key part of this is saving the user time from manual input into their own systems as well as quick input so they can still focus on the game. These aspects should be possible to implament into a dsl however the more natural laguage driven the harder it will be to implament affectivly without significant errors. 
+
 ### Related work
 
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+There are no DSL's in this domain currently. There are phone/web applications, excell, and other stat like sheets that exist. However, most of these things require physical input for every single stat/calculation. They are tedios and take valuable time away form the user. I belive that a DSL could be made that allows for easy stat keeping, veiwing/uploading, and analysis in real time. 
+
```

Comment:
I like this - to check my understanding, it's a way to streamline the input process for stats to aid real-time updates.